Meadowlark-js/tests/e2e/endpoints/educationContents.test.ts

```diff
@@ -64,10 +64,12 @@ describe('Education contents', () => {
 
     it('should edit an education content', async () => {
       const uri = 'uri://ed-fi.org/fake-updated-uri';
+      const id = educationContentLocation.split('/').pop();
       await rootURLRequest()
         .put(educationContentLocation)
         .auth(await getAccessToken('vendor'), { type: 'bearer' })
         .send({
+          id,
```

Comment:
I think these test changes will need rolling back as well

Reply:
@bradbanister , I need the id to compare the body id with the path component. As Stephen mentioned, ODS/API receives the surrogate key, in this case, our surrogate key = documentUuid. Then, if I don't send this value, PUT will return an Error.

Reply:
I'll push a commit in a minute.

The old validation in update I think you are referring to is meaningless now. Previously, it validated that the identity of the document did not change even if other fields did, because we didn't allow that. Now we allow it, but also can't test for that in meadowlark-core anymore. Unfortunately, backends have to do it now via a lookup of meadowlarkId for a given documentUuid.

Reply:
It makes sense. In some point we were replacing the value by documentUuid, so we could update a document using a complete different body, but if allowIdentityUpdates is false, we are filtering the replace with documentUuid and meadowlarkId.